collect_app/src/main/java/org/odk/collect/android/utilities/CustomTabHelper.java

```diff
@@ -93,8 +96,10 @@ public void openUri(Context context, Uri uri) {
             customTabsIntent.intent.setPackage(getPackageName(context).get(0));
             customTabsIntent.launchUrl(context, uri);
         } else {
-            //open in an external browser
```

Comment:
As suggested by @grzesiek2010 and @mmarciniak90, please use an external browser when available.